main.go

```diff
@@ -204,10 +204,26 @@ func startOIDCServer(serviceCache cache.Cache) {
 			Cache: serviceCache,
 		})
 
-		logger.Info("starting oidc service", "port", oidcHTTPPort)
```

Comment:
Serving TLS is ruled by the existence of the certificate files. If I do some mistake providing the certificates, authorino will serve plan HTTP instead of failing. Not a big issue, though.

Ideally, I would control serving plain or TLS based HTTP using a env var pointing to the certificate file(s). If the env var does not exist, serve plain HTTP. If it exists, try to serve TLS and the service will fail if referred cert files are not found or not valid.

Reply:
Adding the env var to control is no big deal. I'll send another PR.
Thanks for suggesting, @eguzki!

Reply:
It was just food for thought for future PR's :)